packages/aws-cdk/lib/init-templates/app/csharp/dotnet.template.bat

```diff
@@ -0,0 +1,8 @@
+@echo off
```

Comment:
should the project readme mention the purpose / usage of this file?
might be helpful for users to understand the implications or removing it / better understanding what we're creating on their behalf